src/spac/spatial_analysis.py

```diff
@@ -251,6 +280,79 @@ def perform_analysis(
 
         return ax
 
+    def get_labels(

```

Comment:
@ruiheesi , please add a doc string for this function

Reply:
Hi George, I have addressed in the new commit, I will close this conversation and please review the latest content. Thanks